Sail/src/Sail/entities/Entity.h

```diff
@@ -9,19 +9,23 @@
 class Entity {
 public:
 	typedef std::shared_ptr<Entity> SPtr;
-	static SPtr Create();
+	static SPtr Create(const std::string& name = "");
 public:
 	Entity();
+	Entity(const std::string& name = "");
 	virtual ~Entity();
 
 	template<typename T, typename... Targs>
 	T* addComponent(Targs... args);
 	template<typename T>
 	T* getComponent();
 	
+	void setName(const std::string& name);
+	const std::string getName();
```

Comment:
```suggestion
	const std::string& getName() const;
```
Added missing qualifiers